src/main/java/tds/exam/results/validation/impl/XSDBackedTDSReportValidator.java

```diff
@@ -11,37 +12,38 @@
 import javax.xml.validation.Schema;
 import javax.xml.validation.SchemaFactory;
 import javax.xml.validation.Validator;
-import java.io.File;
+import java.io.IOException;
 
+import tds.exam.results.configuration.ExamResultsTransmitterServiceProperties;
 import tds.exam.results.trt.TDSReport;
 import tds.exam.results.validation.TDSReportValidator;
 
 @Component
 public class XSDBackedTDSReportValidator implements TDSReportValidator {
     private final JAXBContext jaxbContext;
     private final Validator validator;
+    private final ExamResultsTransmitterServiceProperties properties;
 
-    @Autowired
-    public XSDBackedTDSReportValidator(final JAXBContext jaxbContext) {
+    XSDBackedTDSReportValidator(final JAXBContext jaxbContext,
+                                @Value("classpath:xsd/TestResultsTransmissionFormat_Schema.xsd") final Resource xsd,
```

Comment:
This looks like the spring-standard way to inject a resource.  Option 2 would be to inject a ResourceLoader and load it manually rather than using the @Value annotation... but that feels equally awkward.  Out of curiosity, is the "xsd/" prefix needed or can spring find it by filename alone?